package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "blpapi": "^1.0.0",
-    "bluebird": "^2.8.2",
+    "bluebird": "^2.9.13",
```

Comment:
It may be worthwhile to mention in the commit message the reason for the bump of `bluebird`.  An alternative would be to break down this PR before merging.  For example:

1. Dependency updates
2. The testing infrastructure test cases (that pass)
3. Tests with updated source code in `lib/` to address found issues.